pkg/apis/migration/v1alpha1/model.go

```diff
@@ -131,6 +132,38 @@ func GetStorage(client k8sclient.Client, ref *kapi.ObjectReference) (*MigStorage
 	return &object, err
 }
 
+// Get a referenced Migration for DVM.
+// Return nil if the reference cannot be resolved.
+func GetMigrationForDVM(client k8sclient.Client, owners []v1.OwnerReference) (*MigMigration, error) {
+	if len(owners) == 0 {
+		return nil, nil
+	}
+	migrationName := owners[0].Name
+	migrationObject := MigMigration{}
+	err := client.Get(context.TODO(),
+		k8sclient.ObjectKey{
+			Name:      migrationName,
+			Namespace: OpenshiftMigrationNamespace,
+		}, &migrationObject)
+	//objectList := MigMigrationList{}
```

Comment:
Let's remove these comments if the code is unused.

Reply:
Done